lib/metasploit/framework/data_service/remote/http/remote_host_data_service.rb

```diff
@@ -32,4 +33,20 @@ def update_host(opts)
   def delete_host(opts)
     json_to_mdm_object(self.delete_data(HOST_API_PATH, opts), HOST_MDM_CLASS)
   end
-end
\ No newline at end of file
+
+  def get_host_tags(opts)
+    path = get_path_select(opts, HOST_API_PATH) + '/tag'
+    json_to_mdm_object(self.get_data(path, opts, nil), TAG_MDM_CLASS)
+  end
+
+  def add_host_tag(opts)
+    path = get_path_select(opts, HOST_API_PATH) + '/tag'
+    json_to_mdm_object(self.post_data(path, opts, nil), HOST_MDM_CLASS)
```

Comment:
Why does this method call use `HOST_MDM_CLASS ` when `get_host_tags` uses `TAG_MDM_CLASS `?

Reply:
Have changed `HOST_MDM_CLASS` to `TAG_MDM_CLASS` and started returning tags after additions & deletions to maintain consistency with the rest of the API.